source/_integrations/eway.markdown

```
@@ -0,0 +1,193 @@
+  - Frequency (Hz)
+  - Temperature (°C)
+  - Today's energy generation (kWh)
+  - Total energy generation (kWh)
+
+## Sensors
+
+The integration provides the following sensors:
+
+- **Generation Power**: Current power output (W)
+- **Grid Voltage**: Grid voltage (V)
+- **Input Voltage**: Input voltage from solar 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)